src/XIVLauncher/Windows/SettingsControl.xaml.cs

```diff
@@ -389,6 +390,84 @@ private void GamePathEntry_OnTextChanged(object sender, TextChangedEventArgs e)
             }
         }
 
+        private void ImportConfigButton_OnClick(object sender, RoutedEventArgs e)
+        {
+            var selectConfImportDialog = new Microsoft.Win32.OpenFileDialog();
+            selectConfImportDialog.InitialDirectory = Paths.RoamingPath;
+            selectConfImportDialog.DefaultExt = ".xlconf";
+            selectConfImportDialog.Filter = "XLConfig Export|*.xlconf";
+
+            // Show open file dialog box
+            bool? result = selectConfImportDialog.ShowDialog();
+
+            // Process open file dialog box results
+            if (result == true)
+            {
+                try
+                {
+                    // Open document
+                    string zipFile = selectConfImportDialog.FileName;
+
+                    //TODO: Localize this text like a good programmer
+                    
+                    var importConfirmPromptResult = CustomMessageBox.Show(Loc.Localize("ConfigImportPrompt1", "XIVLauncher is going to import config from the file you selected.\n\n") + zipFile + Loc.Localize("ConfigImportPrompt2", "\n\nThis will overwrite any files that already exist.\n\nDo you want to proceed?"),
+                        Loc.Localize("ConfigImportPromptTitle", "Import confg file?"), MessageBoxButton.YesNo);
```

Comment:
Typo!

Reply:
`confg` spelling corrected. Was that the only one? Or did I miss another? (Will be included on next commit on this branch)